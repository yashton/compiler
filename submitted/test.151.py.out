(PUNCT "(")
(ID "N")
(PUNCT "=")
(NEWLINE)
(LIT 5)
(PUNCT "]")
(NEWLINE)
(ENDMARKER)
