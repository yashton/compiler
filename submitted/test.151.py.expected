(PUNCT "(")
(ID "N")
(PUNCT "=")
(LIT 5)
(PUNCT "]")
(NEWLINE)
(ENDMARKER)
